### ISSUE TITLE
Wrong trevis settings (default has changed)

### ISSUE THREAD

**Issue by tyyko** [cloudfoundry/cloud_controller_ng#739]
## Issue

Previous default virtualization environment for travis was the 'SUDO-ENABLED' (the required one to run the tests) but now is the 'CONTAINER-BASED' and test will not work on forks.

Source: 
First row of the table on
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

## Steps to Reproduce

Fork the repository and enable travis on the fork.

## Expected result

Travis should work out of the box and tests that runs on the original branch should work also on the forked one.

## Current result

Test fails because are killed by Travis due to 'Tests running in parallel using too many processes or threads (e.g. using the parallel_test gem)'

Source: 
https://docs.travis-ci.com/user/common-build-problems/#My-build-script-is-killed-without-any-error

## Possible Fix

Explicitly specify the virtualization environment on the .travis.yml file. Pull request will follow.

**Comment by thausler786** [comment]
Closing this in favor of #740.
